R/utilities.R

```diff
@@ -1077,3 +1077,65 @@ release2json <- function(release, nameslist){
     return(result)
 
 }
+
+#' Error check imputation range for numeric or integer variables
+#' 
+#' Check that the entered imputation range is within the entered data range. If yes, return
+#' the entered imputation range, which will be used as the imputation range for the call
+#' to the utility function `fillMissing()`. If not, return the data range. 
+#' If the imputation range is NULL, default to the data range. The data range may be NULL,
+#' in which case the stability mechanism will be used to determine the data range and
+#' imputation range.
+#' 
+#' @param imputationRange The imputation range entered by the user
+#' @param rng The data range entered by the user
+#' @param var.type The variable type for the histogram data
+#' 
+#' @return the imputation range that will be used for `fillMissing()`.
+
+checkImputationRange <- function(imputationRange, rng, var.type) {
+    # if no imputation range was entered, return the data range.
+    # (Note: rng may be NULL, in which case stability mechanism will be used)
```

Comment:
don't say stability mechanism will be used here since that's only true if histogram is statistic being used.